specs/logging_implementation.adoc

```diff
@@ -0,0 +1,174 @@
+= Specification for logging implementation
+
+Logs and alerts need to be treated as messages emitted by a component.
+
+== Problem description
```

Comment:
Additionally, the updates to in-flight operations, https://github.com/Tendrl/specifications/issues/55, need to be logged under INFO. The updates themselves should just be messages emitted that have the same structure and appropriate metadata. This metadata includes operation uuids. This implies that the details of the jobs being created also need to be logged. This is where tracing and debugging is addressed. The updates being logged provide an id. This id pertains to a specific job. This job being logged enables end-to-end tracing of the job and it's updates across the deployment. This is very important, since it allows an administrator to filter the logs by id and get all the required details.